drivers/volume/portworx/portworx.go

```diff
@@ -5011,3 +5012,43 @@ func (d *portworx) GetAlertsUsingResourceTypeByTime(resourceType api.ResourceTyp
 
 	return alertsResp, nil
 }
+
+// AddCloudDrive add cloud drives to the node as journal using PXCTL
+func (d *portworx) AddCloudDriveAsJournal(n *node.Node, deviceSpec string, poolID int32) error {
```

Comment:
why do we need 2 functions here ? can we update addDriveAsJournal function instead of calling addDriveAsJournal from AddCloudDriveAsJournal ?

Reply:
Done